frontend/e2e/page-objects/InquiryPage.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from "@playwright/test"
+import HelperBase from "./helperBase"
+
+export default class InquiryPage extends HelperBase {
+  async navigateToInquiriesPage() {
+    await this.page.waitForSelector("p.chakra-text.css-0", { timeout: 5000 })
+    await expect(this.page.locator("p.chakra-text.css-0")).toHaveText(
+      "Welcome back, nice to see you again!",
+    )
+    await this.page.getByText("Inquiries").click()
+  }
+
+  async openInquiryForm() {
+    const inquiryTitle = this.page.getByRole("heading", {
+      name: "Inquiries Management",
+    })
+    await expect(inquiryTitle).toHaveText("Inquiries Management")
+    await this.page.getByRole("button", { name: "Inquiry" }).click()
+    await this.page.waitForSelector(".chakra-modal__footer", {
+      state: "visible",
+    })
+    await expect(
+      this.page.getByLabel("Add Inquiry").getByText("Add Inquiry"),
+    ).toHaveText("Add Inquiry")
+  }
+
+  async submitInquiry() {
+    const randomQuestion = this.getRandomQuestion()
+    await this.navigateToInquiriesPage()
+    await this.openInquiryForm()
+    await this.fillTextByTestId("add-inquiry-text", randomQuestion)
+    await this.assertButtonTextByTestIdAndClick("submit-add-inquiry", "Save")
+    await this.waitForNumberOfSeconds(3)
```

Comment:
this tends to be another smell in automated tests. When possible, try to avoid sleeps. (since when executed on different machines with potentially different CPU loads, the sleep can legitimately time out.) If Playwright has some affordance to "waitUntil" or something like that, I'd use that instead.

Reply:
Thank you for the feedback, I’ve removed the fixed sleep (waitForNumberOfSeconds) and replaced it with a more reliable method to wait for the inquiry to appear, instead using fixed time i now wait for selector to be visible